styles/components/list.css

```diff
@@ -0,0 +1,38 @@
+/*unordered list styles*/
+.ul-circle{
+    list-style-type: circle;
+}
+
+.ul-disc{
+    list-style-type: disc;
+}
+
+.ul-square{
+    list-style-type: square;
+}
+
+.ul-no-style{
+    list-style-type: none;
+}
+
+.li-no-style{
+    margin: 1rem 0;

```

Comment:
is this intended?

Reply:
yes it is. i'll move this to utility. Thanks for pointing it out.